recipes/ipython/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "4.1.2" %}
+
+package:
+  name: ipython
+  version: {{ version }}
+
+source:
+  fn: ipython-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipython/ipython-{{ version }}.tar.gz
+  md5: 28c9ebd1abfb9b4a07cb87005f285edd
+
+build:
+  script: python setup.py install
+  number: 0
+  entry_points:
+    - ipython = IPython:start_ipython
+    - ipython2 = IPython:start_ipython  # [py2k]
+    - ipython3 = IPython:start_ipython  # [py3k]
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - pickleshare
+    - simplegeneric >0.8
+# These are listed in the setup.py, but do not cause the build tests to fail.
+# In order to enable these dependencies, additional tests will need to be defined.
```

Comment:
Aha! Sorry, should have been paying attention to the release, not master.
At least they are nicely teed up for next release :wink: